documentation/docs/20-setup/20-local-setup.md

```diff
@@ -108,6 +108,25 @@ It will open a file with your MCP servers where you can add the following config
 }
 ```
 
+## Zed
+
+- Open the command palette (CMD/Ctrl + Shift + P)
```

Comment:
```suggestion
- Open the command palette
```
I'm gonna just remove the command shortcut for consistency with the other sections (and also because if someone configured it differently, it might get confusing)